### DISCUSSION TITLE
Read Structure Result from a River Model

### DISCUSSION THREAD

**Discussion by hawklorry** [DHI/mikeio1d#39] [Q&A] [answered]
I'm wondering how to read the result of structures in a river model. As shown below, there are some structures in the model. They are not included in the Reaches list (only rivers are included in the list). Any thoughts?

![image](https://user-images.githubusercontent.com/3594975/205946857-17358e5f-f66e-420c-990d-f759926b18e6.png)

**Comment by JesperGr** [comment]
Results for structures are stored on the reaches, so you need to find the reach with the structure on it.

**Comment by gedaskir** [comment, marked as answer]
Take a look at the answer in [issue #44](https://github.com/DHI/mikeio1d/discussions/44).
